code/game/machinery/computer/dna_console.dm

```diff
@@ -100,850 +129,1050 @@
-			temp_html += buttons
-			temp_html += "<div class='line'><div class='statusLabel'><b>Advanced Injectors:</b></div></div><br>"
-			temp_html += "<div class='statusLine'><a href='?src=[REF(src)];task=add_advinjector;'>New Selection</a></div>"
-			for(var/A in injector_selection)
-				temp_html += "<div class='statusDisplay'><b>[A]</b>"
-				var/list/true_selection = injector_selection[A]
-				temp_html += "<br>"
-				for(var/B in true_selection)
-					var/datum/mutation/human/HM = B
-					var/mutcolor
-					switch(HM.quality)
-						if(POSITIVE)
-							mutcolor = "good"
-						if(MINOR_NEGATIVE)
-							mutcolor = "average"
-						if(NEGATIVE)
-							mutcolor = "bad"
-					temp_html += "<div class='statusLine'><span class='[mutcolor]'>[HM.name] </span>"
-					temp_html += "<a href='?src=[REF(src)];task=remove_from_advinjector;injector=[A];path=[HM.type];'>Remove</a></div>"
-				if(injectorready < world.time)
-					temp_html += "<div class='statusLine'> <a href='?src=[REF(src)];task=advinjector;injector=[A];'>Print Advanced Injector</a>"
-				else
-					temp_html += "<div class='statusLine'> <span class='linkOff'>Printer ready in [DisplayTimeText(injectorready - world.time, 1)]</span>"
-				temp_html += "<a href='?src=[REF(src)];task=remove_advinjector;injector=[A];'>Remove Injector</a></div>"
-				temp_html += "<br></div>"
+				injectorready = world.time + INJECTOR_TIMEOUT * 8
+		if("add_adv_mut")
+			// Can only be done from the scanner occupant / genetic sequencer
+			if(!can_modify_occupant())
+				return
 
-		else
-			temp_html += status
-			temp_html += buttons
-			temp_html += "<div class='line'><div class='statusLabel'>Genetic Sequence:</div><br>"
-			if(viable_occupant)
-				if(viable_occupant)
-					for(var/A in get_mutation_list())
-						temp_html += display_inactive_sequence(A)
-					temp_html += "<br>"
-				else
-					temp_html += "----"
-				if(viable_occupant && (current_mutation in get_mutation_list(viable_occupant)))
-					temp_html += display_sequence(current_mutation)
-				temp_html += "</div><br>"
+			var/adv_inj = params["advinj"]
+
+			if(!(adv_inj in injector_selection))
+				return
+
+			if(LAZYLEN(injector_selection[adv_inj]) >= max_injector_mutations)
+				to_chat(usr,"<span class='warning'>Advanced injector mutation storage is full.</span>")
+				return
+
+			var/bref = params["mutref"]
+			var/datum/mutation/human/HM = get_mut_by_ref(bref)
+
+			if(!HM)
+				return
+
+			var/instability_total = HM.instability
+
+			for(var/datum/mutation/human/I in injector_selection[adv_inj])
+				instability_total += I.instability * GET_MUTATION_STABILIZER(I)
+
+			if(instability_total > max_injector_instability)
+				to_chat(usr,"<span class='warning'>Advanced injector mutations too instable.</span>")
+				return
+
+			var/datum/mutation/human/A = new HM.type()
+			A.copy_mutation(HM)
+			injector_selection[adv_inj] += A
+			to_chat(usr,"<span class='notice'>Mutation succesfully added to advanced injector.</span>")
```

Comment:
successfully

Reply:
Found all instances of succes and have successfully essed them up. Will be updating as part of larger code documentation I'm currently doing.